<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.transaction.interceptor.RollbackRuleTests" tests="9" skipped="0" failures="0" errors="0" timestamp="2020-04-15T14:22:32" hostname="PandaroiddeiMac.local" time="0.004">
  <properties/>
  <testcase name="ctorArgMustBeAThrowableClassWithNonThrowableType()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <testcase name="ancestry()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <testcase name="alwaysTrueForThrowable()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="foundImmediatelyWithString()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <testcase name="ctorArgExceptionStringNameVersionWithNull()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <testcase name="foundEnclosedExceptionWithEnclosingException()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <testcase name="ctorArgMustBeAThrowableClassWithNullThrowableType()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <testcase name="foundImmediatelyWithClass()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <testcase name="notFound()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
